### ISSUE TITLE
Accessibility: Unable to trigger "shift+alt+f1" for Open Accessibility Help from terminal

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#26103]
Testing -186217

Platform: Linux Mint
Reader: Orca
